components/financial_assistance/app/domain/financial_assistance/operations/applications/medicaid_gateway/run_periodic_data_matching.rb

```diff
@@ -164,13 +164,15 @@ def fetch_eligibility(applicant)
 
           def process_mec_check(application, params)
             @logger.info "process mec_check for determined application #{application.id}"
-            return if params[:skip_mec_call]
+            return true if params[:dry_run_mec]
```

Comment:
Understanding the purpose of this param is to have the ability to generate the report without processing the families for the purposes of dry_run testing. It would be better at line 86 with the param name as mec_check/process_families as boolean which if true will process the families else will not. Either way, you will always generate the report.

Reply:
Yes that was the intention. The `dry_run` is similar to `skip_call` which doesn't execute the MEC call.

Reply:
Yes, the param name `skip_mec_call` is still fine but it should just be at line 86 instead of inside the method.

Reply:
agreed! the code is updated a while ago. Clearing the review to move forward.